Artem.Arkhytiuk/Lesson2/index.css

```diff
@@ -0,0 +1,502 @@
+/*! normalize.css v8.0.0 | MIT License | github.com/necolas/normalize.css */
```

Comment:
You don't make normalize.css copy paste to your index.css.
Imagine if a new version of normalize will appear, what would you do? Yes, you'll need and copy and paste the whole thing again.
You don't need to mix them together it is also a lot of clutter that should not bother you from your own CSS that you writing.
So you better should import it using URL from some of the CDNs (content delivery networks).
It will help to cache it, if, for example, other website uses the same URL which is very common, another guy which will visit your site will not fetch it once again and will use it from the cache immediately.

Consider using this one for importing it.
https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.0/normalize.min.css